lib/pages/main/tab_wallet_contents.dart

```diff
@@ -333,8 +335,8 @@ class _TabWalletContentsState extends State<TabWalletContents> {
                                       vertical:
                                           ThemePaddings.normalPadding / 2),
                                   child: AccountListItem(
-                                      item: appStore.currentQubicIDs[index])));
-                        }, childCount: appStore.currentQubicIDs.length),
+                                      item: appStore.currentQubicIDs[0])));
```

Comment:
Is this intended? or was used for your testing?

Reply:
Thank you for catching that! it was used only for testing.
I'll remove it and push an update shortly.